Src/AutoMoq/AutoMockPropertiesCommand.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using Moq;
+using Ploeh.AutoFixture.Kernel;
+
+namespace Ploeh.AutoFixture.AutoMoq
+{
+    /// <summary>
+    /// A command that populates all public writable properties/fields of a mock object with anonymous values.
+    /// </summary>
+    public class AutoMockPropertiesCommand : ISpecimenCommand
```

Comment:
FWIW, I don't consider it a POLA violation that `fixture.Create<Mock<IHasProperties>>()` returns a differently configured object than `new Mock<IHasProperties>()`... or, if it is, it's also a POLA violation that `fixture.Create<Foo>()` returns a differently populated object than `new Foo()` in all the cases when `Foo` has writable properties.

> While doing this change, a few integration tests started failing. I realized there's a problem with the POC. If this line in the test fixture.Create<IHasProperties>() is changed to fixture.Create<Mock<IHasProperties>>().Object, then the test fails.

This point is, I think, a completely valid reason for doing something more complex than the POC. I hadn't noticed that this was the case with the POC, but it's obviously a flaw in the POC.